__docs__/wonder-blocks-modal/modal-launcher.argtypes.ts

```diff
@@ -3,15 +3,8 @@ import type {ArgTypes} from "@storybook/react-vite";
 export default {
     modal: {
         control: {type: undefined},
-        description: `The modal to render. The modal will be rendered inside
-            of a container whose parent is document.body. This allows us to
-            use ModalLauncher within menus and other components that clip
-            their content. If the modal needs to close itself by some other
-            means than tapping the backdrop or the default close button a
-            render callback can be passed. The closeModal function provided
-            to this callback can be called to close the modal.
-            **Note**: Don't call \`closeModal\` while rendering! It should
-            be used to respond to user intearction, like \`onClick\`.`,
+        description:
```

Comment:
Could the prop description be auto-generated based on the jsdocs for the props type?

Reply:
I don't believe so, because of the `withActionScheduler` HOC wrapping it. We had the same issue with `DrawerLauncher`!

Reply:
@marcysutton I think this could be solved by replacing `withActionScheduler` with the equivalent hook.

Reply:
I added the hook, but the modal prop still requires its description to be derived from the argtypes for some reason. Going to punt on it for now!